recipe/meta.yaml

```
@@ -35,13 +41,13 @@ test:
     - if not exist %LIBRARY_PREFIX%\bin\libx265.dll exit 1        # [win]
```

```suggestion
    - if not exist %LIBRARY_PREFIX%\bin\libx265.dll exit 1        # [win]
    - if not exist %LIBRARY_PREFIX%\bin\x265.exe exit 1          # [win]
```
It looks like `x265.exe` is also available in `%LIBRARY_PREFIX%\bin`, see https://conda-metadata-app.streamlit.app/?q=conda-forge%2Fwin-64%2Fx265-3.5-h2d74725_3.tar.bz2&with_broken=true